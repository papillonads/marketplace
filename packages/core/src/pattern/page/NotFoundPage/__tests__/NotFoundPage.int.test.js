import React from 'react'
import { asyncRender, defaultRender } from './NotFoundPage.int.render'

jest.mock('react-router-dom', () => ({ Link: () => <div>some link</div> }))

describe('<NotFoundPage />', () => {
  afterEach(() => jest.clearAllMocks())

  describe('Snaphot', () => {
    test('must match asyncRender', () => {
      expect(global.renderToJSON(asyncRender)).toMatchSnapshot()
    })

    test('must match defaultRender', () => {
      expect(global.renderToJSON(defaultRender)).toMatchSnapshot()
    })
  })
})
