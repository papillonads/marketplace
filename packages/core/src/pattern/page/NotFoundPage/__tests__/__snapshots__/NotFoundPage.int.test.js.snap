// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NotFoundPage /> Snaphot must match asyncRender 1`] = `null`;

exports[`<NotFoundPage /> Snaphot must match defaultRender 1`] = `
<div>
  <div
    className=""
  >
    <h3>
      No match for 
      fantastic-path-name
    </h3>
  </div>
  Back to 
  <div>
    some link
  </div>
</div>
`;
